dev/keys/keys.c

```diff
@@ -63,5 +63,11 @@ int keys_get_state(uint16_t code)
 		dprintf(INFO, "Invalid keycode requested: %d\n", code);
 		return -1;
 	}
+#if WITH_LK2ND_DEVICE
+#include <lk2nd/keys.h>
```

Comment:
Not sure if it's a good idea to include arbitrary headers within functions, might produce unexpected results...